java/memory/src/main/java/org/apache/arrow/memory/BaseAllocator.java

```diff
@@ -765,7 +765,7 @@ public static Config defaultConfig() {
      */
     @Value.Default
     AllocationManager.Factory getAllocationManagerFactory() {
-      return NettyAllocationManager.FACTORY;
+      return DefaultAllocationManagerOption.DEFAULT_ALLOCATION_MANAGER_FACTORY;
```

Comment:
As the default allocator is changing `RootAllocator` should probably have its defaults[1] changed so that it doesn't generate a default allocator when it wants a netty one.

[1] https://github.com/apache/arrow/blob/dff612e6a468b46b7058757c24c638894101ba67/java/memory/src/main/java/org/apache/arrow/memory/RootAllocator.java#L51

Reply:
ignore me...default is still netty